lib/ui/painting.dart

```diff
@@ -3746,6 +3746,96 @@ class ImageShader extends Shader {
   void _initWithImage(_Image image, int tmx, int tmy, int filterQualityIndex, Float64List matrix4) native 'ImageShader_initWithImage';
 }
 
+/// A shader (as used by [Paint.shader]) that runs provided SPIR-V code.
+///
+/// NOTE: This API is in beta and does not yet work on web.
```

Comment:
per our style guide, we should remove the "NOTE:" prefix.

Reply:
done